tests/functional/services/policy_engine/feeds_data_tests/test_feed_sync.py

```diff
@@ -102,52 +121,172 @@ def test_feeds_get_schema(self, sync_feeds, schema_validator):
         )
 
     @pytest.mark.parametrize(
-        "expected_feed, expected_group", build_feed_sync_test_matrix(), ids=idfn
+        "expected_feed, expected_group, expected_count",
+        build_feed_sync_test_matrix(),
+        ids=idfn,
     )
-    def test_expected_feed_sync(
-        self, expected_feed, expected_group, expected_content, sync_feeds
+    def test_expected_feed_sync_post(
+        self,
+        expected_feed,
+        expected_group,
+        expected_count,
+        expected_content,
+        initial_feed_sync_resp,
+        is_legacy_test,
     ):
-        # sync feeds and verify that the feed was a success
-        feed_sync_resp = sync_feeds
-        assert feed_sync_resp == http_utils.APIResponse(200)
-        assert (
-            self._find_by_attr("feed", feed_sync_resp.body, expected_feed["name"])[
-                "status"
-            ]
-            == "success"
+        """
+        Tests that the post endpoint for initial sync returns expected content for feeds and groups
+        """
+        assert initial_feed_sync_resp == http_utils.APIResponse(200)
+        actual_feed = self._find_by_attr(
+            "feed", initial_feed_sync_resp.body, expected_feed
+        )
+        assert actual_feed is not None
+        assert actual_feed["status"] == "success"
+
+        actual_group = self._find_by_attr(
+            "group", actual_feed["groups"], expected_group
         )
+        assert not isinstance(actual_group, type(None))
+        assert actual_group["status"] == "success"
+        assert actual_group["updated_record_count"] == expected_count
+        assert actual_group["total_time_seconds"] > 0
+
+    @pytest.mark.parametrize(
+        "expected_feed, expected_group, expected_count",
+        build_feed_sync_test_matrix(),
+        ids=idfn,
+    )
+    def test_expected_feed_sync_get_feeds(
```

Comment:
these tests are getting sort of complex. i think each test method would benefit from a short docstring that explains the premise of the test.